Tests/UnitTests/Networking/Backend/__Snapshots__/BackendGetCustomerCenterConfigTests/watchOS-testGetCustomerCenterConfig.1.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Hmm we were missing snapshot tests for watchOS?

Reply:
Something must have changed in main because I remember WatchOS tests were using snapshots of iOS13, so someone must have fixed it